recon/src/libp2p.rs

```diff
@@ -64,7 +64,7 @@ pub trait Recon {
 impl Recon for Arc<Mutex<crate::recon::Recon>> {
     type Hash = Sha256a;
 
-    fn insert_key(&mut self, key: &str) {
+    fn insert_key(&mut self, key: &[u8]) {
```

Comment:
Should this take an EventId instead of arbitrary bytes?

Reply:
done